package.json

```diff
@@ -47,7 +47,7 @@
     "build:esm": "rollup -c",
     "build:esm:mock-classnames": "mock_classnames=on rollup -c",
     "build:react-icons": "svg2react-icon --typescript --keep-colors node_modules/monday-ui-style/src/Icons src/components/Icon/Icons",
-    "storybook": "storybook=true start-storybook -p 7007",
+    "storybook": "storybook=true start-storybook -p 7008",
```

Comment:
why?

Reply:
For convenience - it's running on same port a monday-ui-components does, so it makes it uncomfortable to run both packages at the same time

Reply:
I think storybook asks you each time if you want to try a different port if another process already runs on it, no?

Reply:
It asks you and you have to input "y". It's not that a big deal, but still not really convenient. Especially, when it can be easily solved just by changing port on our side

Reply:
ok